### ISSUE TITLE
How to set the clean start property in MQTT 5?

### ISSUE THREAD

**Issue by SeppPenner** [dotnet/MQTTnet#1710]
### Describe your question
I have a request related to Sparkplug. In the Sparkplug IIOT standard, version 3.0, the host application must set the clean start property = `true` and session expiry interval = 0 for MQTT v5.0 on connect.

How to achieve that?

My current code looks like that:

```csharp
var builder = new MqttClientOptionsBuilder()
    .WithClientId("test");

    switch (this.Options.MqttProtocolVersion)
    {
        case MqttProtocolVersion.V311:
            builder.WithCleanSession(true);
            break;
        case MqttProtocolVersion.V500:
            //Todo: Set clean start?!
            builder.WithSessionExpiryInterval(0);
            break;
    }
```

### Which project is your question related to?
- Client

**Comment by chkr1011** [comment]
You need to set clean session to true. With MQTT version 5 this was renamed to clean start.

**Comment by SeppPenner** [comment]
So, this is the same property in MQTT v5 and just the offical name is changed? Ok, good to know. Thank you :)

**Comment by chkr1011** [comment]
@SeppPenner Yes the official name has changed. I will add a new overload to the options builder.
